Sources/EmbraceCoreDataInternal/CoreDataWrapper.swift

```diff
@@ -138,7 +138,9 @@ public class CoreDataWrapper {
             }
 
             do {
-                try context.save()
+                if context.hasChanges {
```

Comment:
Should help fix high disk writes and cpu reported by Apple as well, but not completely.